src/ethereum/info/ethDevInfo.ts

```diff
@@ -129,7 +129,8 @@ const currencyInfo: EdgeCurrencyInfo = {
   currencyCode: 'ETH',
   customFeeTemplate: evmCustomFeeTemplate,
   customTokenTemplate: evmCustomTokenTemplate,
-  displayName: 'Dev Ethereum',
+  assetDisplayName: 'Dev Ethereum',
+  chainDisplayName: 'Dev Ethereum',
```

Comment:
The other Ethereum testnets, like Holesky Testnet, just use "Ethereum" as the asset name. The other non-Ethereum testnets, like Hedera Testnet, work like this one, where they have "Testnet" in the asset name.

I don't know what the correct solution is, or if there is even a problem in the first place, but it is a discrepancy.